sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -310,6 +310,10 @@ private[sql] abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
       case logical.MapPartitionsInR(f, p, b, is, os, objAttr, child) =>
         execution.MapPartitionsExec(
           execution.r.MapPartitionsRWrapper(f, p, b, is, os), objAttr, planLater(child)) :: Nil
+      case logical.MapGroupsR(f, p, b, is, os, key, value, grouping, data, objAttr, child) =>
```

Comment:
Renamed MapGroupsR to MapGroupsPartitionsInR.
Or maybe MapGroupsInR is better. Not sure. @sun-rui ?